Build error on 'make build'

**Describe the bug**
I am following the guide [https://github.com/tkestack/tapp#build](url) to build, but on command `make build` and `make image` getting errors: 
```shell
~/gopath/src/tkestack.io/tapp]# make build
hack/build.sh
hack/build.sh: line 20: cd: /root/gopath/src/git.tencent.com/tke/tapp-controller: No such file or directory
make: *** [build] Error 1
```
```shell
~/gopath/src/tkestack.io/tapp]# make image
hack/verify-gofmt.sh
hack/build-image.sh
hack/build-image.sh: line 23: cd: /root/gopath/src/git.tencent.com/tke/tapp-controller: No such file or directory
make: *** [image] Error 1
```

**Environment**
 - OS: Linux VM_149_11_centos 3.10.107-1-tlinux2_kvm_guest-0049 #1 SMP Tue Jul 30 23:46:29 CST 2019 x86_64 x86_64 x86_64 GNU/Linux
 - golang: go version go1.12.4 linux/amd64

--- a/hack/build-image.sh
+++ b/hack/build-image.sh
@@ -20,7 +20,7 @@ fi
 
 go get github.com/mitchellh/gox
 
-cd $GOPATH/src/$API_GO_PACKAGE/tapp-controller
+cd $GOPATH/src/$API_GO_PACKAGE/tapp
 CGO_ENABLED=0 gox -osarch="linux/amd64" -ldflags "$(api::version::ldflags)" \
 	-output=$BIN
 docker build build/docker/ -t tappcontroller:v1.0.0

--- a/hack/build.sh
+++ b/hack/build.sh
@@ -17,7 +17,7 @@ else
   api::version::save_version_vars "${GIT_VERSION_FILE}"
 fi
 
-cd $GOPATH/src/$API_GO_PACKAGE/tapp-controller
+cd $GOPATH/src/$API_GO_PACKAGE/tapp
 go build -o bin/tapp-controller -ldflags "$(api::version::ldflags)" .
 if [ $? -eq 0 ]; then
   echo "Build success!"

--- a/hack/lib/golang.sh
+++ b/hack/lib/golang.sh
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-readonly API_GO_PACKAGE=git.tencent.com/tke
+readonly API_GO_PACKAGE=tkestack.io
 readonly API_GOPATH="${ROOT}/go"

